cmd/manager/main.go

```diff
@@ -31,7 +31,7 @@ import (
 
 func main() {
 	klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
-	klog.InitFlags(klogFlags)
+	klog.InitFlags(nil)
 	flag.Parse()
```

Comment:
any reason this need to be different to https://github.com/kubernetes-sigs/cluster-api/blob/master/cmd/manager/main.go#L34

Reply:
Yes, we do. Otherwise, `--logtostderr` will not get passed properly. I tried various combinations removing and adding lines of the code between 34-42. This was the only functional solution.

Reply:
it seems to be set here https://github.com/kubernetes-sigs/cluster-api/blob/master/cmd/manager/main.go#L34
We might need to update the flags we set at runtime according to the new code

Reply:
That's set from within the code. There are other flags we set, e.g. `--v` which is not present without this PR and not set inside the manager.